src/helix/mod.rs

```diff
@@ -360,19 +264,59 @@ pub trait RequestPost: Request {
         serde_json::to_string(body)
     }
 
+    /// Get Request to use in your client
+    fn create_request(
+        &self,
+        body: Self::Body,
+        token: &str,
+        client_id: &str,
+    ) -> Result<http::Request<Vec<u8>>, CreateRequestError>
+    {
+        let uri = self.get_uri()?;
+
+        let body = self.body(&body)?;
+        // eprintln!("\n\nbody is ------------ {} ------------", body);
+
+        let mut bearer =
+            http::HeaderValue::from_str(&format!("Bearer {}", token)).map_err(|_| {
+                CreateRequestError::Custom("Could not make token into headervalue".into())
+            })?;
+        bearer.set_sensitive(true);
+        http::Request::builder()
+            .method(http::Method::POST)
+            .uri(uri)
+            .header("Client-ID", client_id)
+            .header("Content-Type", "application/json")
+            .header(http::header::AUTHORIZATION, bearer)
+            .body(body.into_bytes())
+            .map_err(Into::into)
+    }
+
     /// Parse response. Override for different behavior
-    fn result<RE>(
+    fn parse_response(
         self,
-        _url: &url::Url,
-        _body: &str,
+        uri: &http::Uri,
```

Comment:
Is this needed? Do we need to report the `Uri` in the error message? 

I think having it there is a great benefit.

Also, we don't use self.get_uri() because I want to figure out first if this should be a method on self or not.